models/_sector/nft/trades/nft_events.sql

```diff
@@ -1,7 +1,8 @@
 {{ config(
     schema = 'nft',
     alias = alias('events'),
-    partition_by = ['block_date'],
+    tags = ['dunesql'],
+    partition_by = ['blockchain','project','block_month'],
```

Comment:
could this granular of a partition lead to v small partitions?
can validate with a simple query to group by these fields and get a count, order by asc?

Reply:
we still have a backlog item to build a universal partition strategy post-migration, but i'd rather lean towards too big of partitions vs. too small in the short term

Reply:
to be fair, a universal `block_month` on all dex models likely led to many small partitions on smaller projects too

Reply:
(these comments apply throughout the PR)

Reply:
This is current trade-off I saw as best.
It's speed for cross-chain queries vs speed for blockchain/marketplace specific queries.
You don't really want a query on only blur on eth mainnet to have to scan all the magiceden solana trades. And since most of the uses are still project/blockchain specific I went with this partitioning.

Reply:
i'm good with that 👍 
we can revisit when we get to the future backlog item, just to ensure consistency across sector spells and such